packages/server/src/fhir/asyncjob.ts

```diff
@@ -0,0 +1,43 @@
+import { AsyncJob, Bundle, BundleEntry, BundleEntryResponse } from '@medplum/fhirtypes';
+import { Request, Response, Router } from 'express';
+import { asyncWrap } from '../async';
+import { Repository } from './repo';
+import { allOk } from '@medplum/core';
+import { sendResponse } from './routes';
+
+// Asychronous Job Status API
+// https://hl7.org/fhir/async-bundle.html
+
+export const asyncJobRouter = Router();
+
+asyncJobRouter.get(
+  '/:id/status',
+  asyncWrap(async (req: Request, res: Response) => {
+    const { id } = req.params;
+    const repo = res.locals.repo as Repository;
+    const asyncJob = await repo.readResource<AsyncJob>('AsyncJob', id);
+
+    if (asyncJob.status !== 'completed') {
+      res.status(202).end();
+      return;
+    }
+
+    const entry: BundleEntry[] = [];
```

Comment:
If I'm reading all of this correctly, we can just create the JSON object directly.

I.e.,

```ts
    await sendResponse(res, allOk, {
      resourceType: 'Bundle',
      type: 'batch-response',
      entry: [{
        status: '200 OK',
        location: asyncJob.request,
      }]
    } as Bundle);
```

Reply:
That works! The JSON object would be this:
```
    await sendResponse(res, allOk, {
      resourceType: 'Bundle',
      type: 'batch-response',
      entry: [
        {
          response: {
            status: '200 OK',
            location: asyncJob.request,
          },
        },
      ],
    } as Bundle);
    ```